Draft.md

```diff
@@ -174,13 +179,13 @@ Bei der Beurteilung der wissenschaftlichen Qualität von Forschungssoftware sind
 
 * **Wie funktioniert der wissenschaftliche Arbeitsablauf, der in der Software implementiert wurde?** Hier geht es weniger um die konkrete technische Implementierung, sondern vielmehr um die generelle Methodik. Welche wesentlichen Schritte durchlaufen Daten, um eine bestimmte Aufgabe zu lösen? Welche statistischen Werkzeuge kommen zum Einsatz? Gibt es Vergleichsdaten? Ein Beispiel für eine mögliche Prozessierung ist die Bereinigung von Eingabedaten mittels eines Referenzdatensatzes, um eine Klassifizierung mit einer hierarchischen Clusteranalyse zu erstellen und zuletzt als paarweise Distanzmatrix zu visualisieren.
 
-* **Ist die Behauptung, eine bestimmte wissenschaftliche Fragestellung mit dem gewähltem Arbeitsablauf beantworten zu können, korrekt?** Forschungssoftware verspricht meist – implizit oder explizit – die Beantwortung wissenschaftlicher Fragestellungen zu ermöglichen oder zumindest zu vereinfachen. Eine GIS-Applikation zur Analyse von Punktmustern kann etwa für den Zweck konzipiert sein menschliches Siedlungsverhalten aufzuzeigen. Die Frage, ob der angewendete Algorithmus dazu überhaupt in der Lage ist, kann den Umfang einer Rezension bei weitem übersteigen. Dennoch sollte der Rezensent versuchen eine Einschätzung der Plausibilität vorzunehmen oder zumindest das Problem der Einschätzbarkeit selbst offenzulegen, um die Leser zu sensibilisieren. Zur Beurteilung kann es helfen sich zu vergegenwärtigen, welche Schlüsse aus den rohen, unverarbeiteten Eingabedaten überhaupt theoretisch ableitbar sein könnten.
```

Comment:
Auf einen der Absätze hier in 'Einsatz in der Archäologie und wissenschaftlicher Zweck' bezieht sich:

> Einsatz in der Archäologie: Der entsprechende Passus berührt die Frage nach Effizienz vs. Effektivität. Das Verwenden dieses Begriffspaares würde die Klarheit der dargelegten Überlegungen vielleicht noch unterstützten?

(Mir ist da nicht ganz klar, welcher Absatz gemeint ist)

Reply:
Ich glaube nicht, dass es uns hier um Effizienz vs. Effekitivität geht. Ist mit Effektivität gemeint "Dies ist zum einen – das ist für Forschungssoftware vorrangig – die Frage, ob die Software einen sinnvollen Beitrag zur Bearbeitung der archäologischen Fragestellung liefert."? und Mit Effizienz "Zum anderen ist zu beurteilen, ob Software eine korrekte Umsetzung der angestrebten Arbeit leistet. Beide Fragen sind mitunter nicht leicht zu beantworten."? Aber dann wäre doch eher "Relevanz" und "Korrektheit" das Paar und ein "vs" sehe ich auch nicht. Also ich bin ratlos.

Reply:
Muss eben nochmal die Defintiionen vor Augen haben: 
Effektivität: Das Richtige tun. - ein Programm, das macht was wir brauchen.
Effizienz: Es richtig tun. - ein Programm, das das was wir brauchen, richtig / gut / schnell tut.

"sinnvoller Beitrag" ist damit die Effektivität und "korrekte Umsetzung" ist Effizienz.

Oder?
Mir persönlich würde die Nutzung dieser beiden Begriffe allerdings beim Lesen des Beitrages vermutlich nicht weiterhelfen.

Reply:
dito, geht mir auch so.

Reply:
Ja, ich hatte ja mit der Anmerkung auch schon Schwierigkeiten. Würde sagen: 3:1 -> wir verzichten auf die Einarbeitung dieses Begriffpaares.